etl/steps/data/grapher/countries/2023-10-01/gleditsch.meta.yml

```diff
@@ -3,5 +3,5 @@ tables:
     variables:
       is_present:
         title: Sovereign state
-        description_short: Whether the country is present in the Gleditsch and Ward countries dataset.
+        description_short: The country is identified as a sovereign state by Gleditsch and Ward dataset.
```

Comment:
same here - e.g.,  `The identification of a country as a sovereign state by Gleditsch and Ward depends on its autonomous territorial administration and recognition of independence by both local actors and other states.`